src/odemis/util/driver.py

```
@@ -117,6 +118,67 @@ def readMemoryUsage():
+    try:
+        # Get list of sessions
+        sessions = subprocess.check_output(
+            ["loginctl", "list-sessions", "--no-legend"], text=True
+        ).strip().splitlines()
+
+        for session in sessions:
+            parts = session.split()
+            if len(parts) >= 3:
+                session_id, user = parts[0], parts[2]
+                # Check session type (wayland or x11)
+                stype = subprocess.check_output(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)